src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcessDumpUtility.cs

```diff
@@ -100,27 +99,38 @@ private static string BuildProcDumpArgs(int processId, string filename)
             // -accepteula: Auto accept end-user license agreement
             // -t: Write a dump when the process terminates.
             // This will create a minidump of the process with specified filename
-            return "-accepteula -t " + processId + " " + filename + ".dmp";
+            return "-accepteula -e 1 -g -t -f STACK_OVERFLOW -f ACCESS_VIOLATION " + processId + " " + filename + ".dmp";
```

Comment:
Can you write in detail in description what validations you did for these parameters check?

Reply:
Updated the description.